job_templates/run_command_-_grubby_reprovision.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+kind: job_template
+name: Run Command - Grubby Reprovision
+model: JobTemplate
+job_category: Power
+provider_type: SSH
+%>
+<%
+  if @host.build?
+-%>
+KERNEL="/boot/grubby-reporovision-kernel"
+INITRD="/boot/grubby-reporovision-initrd.img"
+wget -O "$KERNEL" "<%= @host.url_for_boot(:kernel) %>"
+wget -O "$INITRD" "<%= @host.url_for_boot(:initrd) %>"
+grubby --add-kernel=$KERNEL --initrd=$INITRD --copy-default \
```

Comment:
It can be missing, but do you really want me to put "if os is redhat then yum install" stuff? I mean, it's easier to just error out and user will immediately see this from the result and reschedule this.

Reply:
there's a snippet for installing a package ;-) Just use this

```
<%= render_template('Package Action - SSH Default', :action => 'install', :package => 'grubby') %>
```

as a line above this. This will install `grubby` package using yum, apt and zypper